### DISCUSSION TITLE
Calendar: Add highlight during range selection

### DISCUSSION THREAD

**Discussion by AnTiiK** [primefaces/community#1633] [PrimeNG]
To improve the calendar range selection UX, I would like to highlight the preview of the selection (Like Angular Material)
Currently, it's not possible to customize this into the actual component.
If needed, I can make a PR with this feature

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the feedback, we'll wait for the upvotes to implement this as a feature.
